clients/imodels-client-management/package.json

```diff
@@ -19,10 +19,11 @@
     "name": "Bentley Systems, Inc.",
     "url": "http://www.bentley.com"
   },
-  "main": "lib/index.js",
-  "typings": "lib/index",
+  "main": "./lib/cjs/index.js",
+  "module": "./lib/esm/index.js",
```

Comment:
as per [this discussion on Stack Overflow](https://stackoverflow.com/questions/42708484/what-is-the-module-package-json-field-for) `exports` field should be preferred. Should we switch to `exports`? if not, why?